wsd/Admin.cpp

```diff
@@ -750,7 +756,7 @@ void Admin::triggerMemoryCleanup(const size_t totalMem)
             memLimit << "% (" << static_cast<size_t>(_totalSysMemKb * memLimit / 100.) << " KB).");
 
     const double memToFreePercentage = (totalMem / static_cast<double>(_totalSysMemKb)) - memLimit / 100.;
-    int memToFreeKb = static_cast<int>(memToFreePercentage > 0.0 ? memToFreePercentage * _totalSysMemKb : 0);
+    int memToFreeKb = clamp<double>(memToFreePercentage > 0.0 ? memToFreePercentage * _totalSysMemKb : 0, 0, std::numeric_limits<int>::max());
```

Comment:
If we are clamping, then we don't need the conditional. Should be more readable that way.

Reply:
Done, see https://github.com/CollaboraOnline/online/pull/5472.